pkg/cmd/start/start.go

```diff
@@ -500,7 +500,7 @@ func (o *Options) createLighthouseJob(jobName string, cfg *config.Config) error
 	launchClient := launcher.NewLauncher(o.LHClient, o.Namespace)
 	lhjob, err = launchClient.Launch(lhjob)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create lighthousejob %s in namespace %s", lhjob.Name, ns)
+		return errors.Wrapf(err, "failed to create lighthousejob in namespace %s", ns)
```

Comment:
I would expect the linter to catch this error, something like trying to access fields of a null value. May be we have some linter rule missing? Good catch!

Reply:
I don't think the linter have been run on jx-pipeline...

Reply:
we have a lint pipeline for this repo 🤷